privacy/policies.go

```diff
@@ -0,0 +1,27 @@
+package privacy
+
+import (
+	"github.com/mxmCherry/openrtb"
+
+	"github.com/prebid/prebid-server/privacy/ccpa"
+	"github.com/prebid/prebid-server/privacy/gdpr"
+)
+
+// Policies represents the privacy policies and regulations of an OpenRTB bid request.
+type Policies struct {
+	GDPR gdpr.Policy
+	CCPA ccpa.Policy
+}
+
+// Write mutates an OpenRTB bid request with the context of the policies.
+func (p Policies) Write(req *openrtb.BidRequest) error {
+	if err := p.GDPR.Write(req); err != nil {
+		return err
+	}
+
+	if err := p.CCPA.Write(req); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
I realize both `privacy/ccpa/ccpa.go` and `privacy/gdpr/gdpr.go` have 100% coverage with their respective "_test.go" files but do you think a `privacy/policies_test.go` is also desired?

Reply:
Refactored and added tests.